custom/icds_reports/views.py

```diff
@@ -1766,6 +1785,7 @@ def get(self, request, *args, **kwargs):
         config = {
             'month': tuple(selected_month.timetuple())[:3],
             'aggregation_level': 1,
+            'num_launched_awcs__gt': 1,
```

Comment:
same screenshot question for this kind of filtering

Reply:
yeahing attaching to description